docs/website/layouts/404.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE>
+<html>
+  <head>
+    <title>
+      404 Not Found | Open Policy Agent
+    </title>
+    {{ partial "css.html" . }}
+  </head>
+  <body>
+    <section class="hero is-light is-fullheight">
+      <div class="hero-body has-text-centered">
+        <div class="container">
+          <p class="title is-size-1 is-size-2-mobile has-text-weight-light">
+            404 Not Found
+          </p>
+
+          <div class="content is-medium">
+            <p>
+              We're sorry, but this page could not be found. Perhaps you were looking for one of the following:
+            </p>
+
+            <ul>
```

Comment:
Can we set `list-style: none` or use one of the bulma lists https://bulma.io/documentation/components/list/ to get the bullet points removed from the side?